examples/snippets/modules/ROOT/examples/build-spark-streamlets-scala/build.sbt

```diff
@@ -0,0 +1,79 @@
+lazy val root =
```

Comment:
Was this new project added to the examples.yaml file so we can update versions in it and run targets against it with the examples scripts?

Reply:
Good catch. Maybe we should just list the folders ... :grin: